consensus/ibft/sign.go

```diff
@@ -31,86 +41,72 @@ func ecrecoverImpl(sig, msg []byte) (types.Address, error) {
 }
 
 func ecrecoverProposer(h *types.Header) (types.Address, error) {
-	// get the extra part that contains the seal
-	extra, err := getIbftExtra(h)
+	seal, err := unpackProposerSealFromIbftExtra(h)
 	if err != nil {
 		return types.Address{}, err
 	}
 
-	// Calculate the header hash (keccak of RLP)
-	hash, err := calculateHeaderHash(h)
+	// get the sig
+	msg, err := calculateHeaderHash(h)
```

Comment:
Why do we need to have both calculateHeaderHash and istanbulHeaderHash if they do same thing?

Reply:
I think istanbulHeaderHash has been removed in the latest PR